PHPCompatibility/Sniff.php

```diff
@@ -732,20 +732,20 @@ public function isVariable(File $phpcsFile, $start, $end, $targetNestingLevel)
         }
 
         // Ok, so the first variable is at the right level, now are there any
-        // blacklisted tokens within the empty() ?
-        $hasBadToken = $phpcsFile->findNext($tokenBlackList, $start, $end);
+        // blocklisted tokens within the empty() ?
+        $hasBadToken = $phpcsFile->findNext($tokenBlockList, $start, $end);
         if ($hasBadToken === false) {
             return true;
         }
 
-        // If there are also bracket tokens, the blacklisted token might be part of a variable
+        // If there are also bracket tokens, the blocklisted token might be part of a
         // variable, but if there are no bracket tokens, we know we have an error.
```

Comment:
You turned the variable variable into a variable... this was not a typo...

Reply:
Thanks, fixed!